packages/panna-sdk/src/core/utils/utils.ts

```diff
@@ -207,3 +209,78 @@ export const accountBalanceInFiat = async function (
 
   return result;
 };
+
+/**
+ * Get the total fiat value of multiple tokens and individual token balances
+ * @param params - Parameters for getting multiple account balances
+ * @returns Total value and individual token balances with fiat values
+ * @throws Error if address is invalid or if any token fetch fails
+ * @example
+ * ```ts
+ * // Get portfolio value for multiple tokens
+ * const result = await accountBalancesInFiat({
+ *   client: pannaClient,
+ *   address: '0x...',
+ *   chain: lisk,
+ *   tokens: [
+ *     {}, // Native token
+ *     { address: '0x...' }, // USDC
+ *     { address: '0x...' }  // DAI
+ *   ],
+ *   currency: 'USD'
+ * });
+ * // result: {
+ * //   totalValue: { amount: 5250.75, currency: 'USD' },
+ * //   tokenBalances: [...]
+ * // }
+ * ```
+ */
+export async function accountBalancesInFiat(
+  params: AccountBalancesInFiatParams
+): Promise<AccountBalancesInFiatResult> {
+  if (!isValidAddress(params.address)) {
+    throw new Error('Invalid address format');
+  }
+
+  const currency = params.currency || 'USD';
```

Comment:
Let's define a `DEFAULT_CURRENCY` constant (in defaults.ts maybe!?) and use it all across?
Same should be applicable to all the defaults.